app/constants/preferences.ts

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-export const CATEGORIES_TO_KEEP: Record<string, string> = {
```

Comment:
Just curious on why we needed to remove the type definition.

Reply:
Just a QoL improvement. If we have it as a record from string to string, autocomplete won't work, since any string can be a key.

Leaving it as this, `CATEGORIES_TO_KEEP` allows for autocomplete in the editor.